packages/ingest/src/import-lessons/graphql/query.ts

```diff
@@ -6,7 +6,11 @@ export const query = gql`
       limit: $limit
       offset: $offset
       distinct_on: lessonSlug
-      where: { videoTitle: { _is_null: false }, isLegacy: { _eq: false } }
+      where: {
+        videoTitle: { _is_null: false }
+        isLegacy: { _eq: false }
+        subjectSlug: { _neq: "english" }
```

Comment:
This might be a gotcha. Needs a comment?

Reply:
Good point. Eventually it should be a parameter so that whoever is kicking off an ingest can control filters on say subject or key stage

Reply:
I meant to take it out